README.md

```diff
@@ -1,4 +1,51 @@
 # Data-table
-A data-table built with aurelia-orm.
+A data-table with pagination built with [Spoonx/Aurelia-orm](https://github.com/SpoonX/aurelia-orm) and [Spoonx/Aurelia-pager](https://github.com/SpoonX/aurelia-pager)
 
-Still under development.
+### columns
+String representing the column names, supports renaming columns with `as` e.g. `name as username`.
+
+### repository (optional)
+A entity with [Spoonx/Aurelia-orm](https://github.com/SpoonX/aurelia-orm) e.g. `entityManager.getRepository('users')`
+
+### resource (optional)
+The name of a repository e.g. `users`
```

Comment:
Could be explained in more detail. Maybe use what ORM has:

> This tells the component which repository to get. This takes away the code you'd otherwise have to write with repository.bind.